charts/dependentapi-simple-operator/values.yaml

```diff
@@ -1,3 +1,3 @@
-image: mtr.devops.telekom.de/magenta_canvas/public:dependentapi-simple-operator-0.1.0
+image: ankursarkar/dependentapi-simple-operator:0.2.0-rc
```

Comment:
avoid using release-candidate versions "-rc"

Reply:
Will update the image without "-rc"